book/svm.qmd

```diff
@@ -8,234 +8,248 @@ abstract: TODO (150-200 WORDS)
+\forall i = 1,...,n \\
 \end{dcases}
 \end{aligned}
+\label{eq:surv_ssvmvb2}
 $$
 
-and predictions are made with,
+The blue parts of the equation make up the ranking model and the red parts are the regression model.
+$\gamma$ is the penalty associated with the regression method and $\mu$ is the penalty associated with the ranking method setting $\gamma = 0$ results in the regression SVM and $\mu = 0$ results in the ranking SSVM.
+Hence, fitting the hybrid model and tuning these parameters is an efficient way to automatically detect which SSVM is best suited to a given task.
+
+Once the model is fit, a prediction from given features $\xx^* \in \Reals^p$, can be made using the equation below, again with the ranking and regression contributions highlighted in blue and red respectively.
 
 $$
-\hatg(X^*) = \sum^n_{i=1} \textcolor{blue}{\alpha_i(K(X_i, X^*) - K(X_{j(i)}, X^*))} + \textcolor{red}{\alpha^*_i K(X_i, X^*) - \Delta_i\alpha_i'K(X_i, X^*)} + \beta_0
+\hatg(\xx^*) = \sum^n_{i=1} \textcolor{CornflowerBlue}{\alpha_i(K(\xx_i, \xx^*) - K(\XX_{j(i)}, \xx^*))} + \textcolor{Rhodamine}{\alpha^*_i K(\xx_i, \xx^*) - \Delta_i\alpha_i'K(\xx_i, \xx^*)} + \beta_0
 $$
 
-The regularizer hyper-parameters $C$ and $\mu$ now have a clear interpretation. $C$ is the penalty associated with the regression method and $\mu$ is the penalty associated with the ranking method. By always fitting the hybrid models and tuning these two parameters, there is never a requirement to separately fit the regression or ranking methods as these would be automatically identified as superior in the tuning procedure. Moreover, the hybrid model retains the interpretability of the regression method and predictions can be interpreted as survival times. The hybrid method is implemented in $\pkg{survivalsvm}$ as 'hybrid'. By Van Belle's own simulation studies, these models do not outperform the Cox PH with respect to Harrell's C.
+where $\alpha_i, \alpha_i^*, \alpha_i'$ are Lagrange multipliers and $K$ is a chosen kernel function, which may have further hyper-parameters to select or tune.
 
-#### SSVR-MRL {.unnumbered .unlisted}
+## Conclusion
 
-Not all SSVMs can be considered a variant of the SSVM-Hybrid, though all prominent and commonly utilised suggestions do seem to have this formulation. One other algorithm of note is termed here the 'SSVM-MRL'  [@Goli2016a; @Goli2016b], which is a regression SSVM. The algorithm is identical to SVCR with one additional constraint.
+:::: {.callout-warning icon=false}
 
-**SSVR-MRL**\label{mod-ssvrmrl}\\
-$$
-\begin{aligned}
-& \min_{\beta, \beta_0, \xi, \xi^*,\xi'} \frac{1}{2}\|\beta\|^2 + C\sum^n_{i=1} (\xi_i + \xi_i^*) + C^*\sum^n_{i=1} \xi_i' \\
-& \textrm{subject to}
-\begin{dcases}
-& T_i - g(X_i) \leq \xi_i \\
-& \Delta_i(g(X_i) - T_i) \leq \xi_i^* \\
-& (1 - \Delta_i)(g(X_i) - T_i - MRL(T_i|\hatS)) \leq \xi_i' \\
-& \xi_i, \xi_i^*, \xi_i' \geq 0 \\
-& \forall i = 1,...,n
-\end{dcases}
-\end{aligned}
-$$
+## Key takeaways
```

Comment:
maybe also a good place to list limitations, e.g. needs lots of tuning, doesn't make distribution predictions.
On that note, I thought it might make sense to have either at the beginning of each section a code that indicates which types of predictions and/or which survival settings a method is applicable to
Otherwise could have a table for all methods in the beginning of the ML Models section with that information (but it might be boring, because most methods will be able to do all predictions and all settings (in theory))

Reply:
Method or model class? SSVMs are the only ones that make regression predictions and no distribution predictions. Some RSFs make distribution predictions, some make ranking predictions, so how would we show that?

Reply:
yeah, maybe not necessary

Reply:
just say in the beginning of the section or at the end, that SVMs are one of the few model classes without distr prediction